conda.recipe/meta.yaml

```diff
@@ -0,0 +1,21 @@
+package:
+  name: nb_conda_kernels
+  version: "0.1.0"
+
+build:
+  number: {{ environ.get('GIT_DESCRIBE_NUMBER', 0) }}
+  string: py{{ environ.get('PY_VER').replace('.', '') }}_{{ environ.get('GIT_BUILD_STR', '0') }}
+
+requirements:
+  build:
+    - python
+  run:
+    - python
+    - nbwrapper
+
+source:
+  path: ../
+
+about:
+  home: https://github.com/Anaconda-Server/anaconda-notebook
+  license: TBD
```

Comment:
As I said to @ijstokes, we should license with BSD, it is the standard in the pydata ecosystem...